drivers/driver.go

```diff
@@ -13,13 +14,13 @@ var registeredDrivers = make(map[string]Driver)
 type Driver interface {
 	Open(connURL string) (Driver, error)
 	Ping() error
-	CreateSchemaTableIfNotExists() error
 	Close() error
 	Lock() error
 	Unlock() error
 	Apply(migration *models.Migration, saveVersion bool) error
 	AppliedMigrations() ([]*models.Migration, error)
 	SetConfig(string, interface{}) error
+	DB() *sql.DB
```

Comment:
I don't believe this will be necessary. As discussed below, let's remove the option for a driver to create its own DB instance. This reduces complexity and forces the user to always pass a db instance from the upper layer. As a result, I think we can also remove the `Open` method.